### PR TITLE
Login style updates

### LINKED ISSUE #503: Update login page styles
Reference: https://github.com/mozilla/network-pulse/issues/1089

@natalieworth @mmmavis

### LINKED ISSUE #504: Login page fonts not rendering
Current fonts used on the login page are Zilla Slab & Open Sans. They're not being loaded if the font isn't available locally.
Screen Shot 2019-06-04 at 12 32 24 PM

### PR DESCRIPTION
**Pull request by youriwims**
Closes #503 
Closes #504

Login: https://network-pulse-api-stagi-pr-505.herokuapp.com/accounts/login/?next=/api/pulse/v2/entries/
Email Exists: https://network-pulse-api-stagi-pr-505.herokuapp.com/accounts/login/?next=%2Faccounts%2Fsocial%2Fconnections%2F%3Fnext%3D%252Fadmin%252F&promptconnection=True&provider=GitHub

*Dev Note: Will revert[ L6 ](https://github.com/mozilla/network-pulse-api/pull/505/files#diff-782550878371478afc36251b5c4e6435R6) of `./templates/socialaccount/snippets/social-buttons.html` & L24/L26 of `base.html` once all approvals are done.

### CODE REVIEW

**Comment by mmmavis** [comment]
Buttons look a bit off on the test log in page. @natalieworth @youriwims thoughts?

image

**Comment by youriwims** [comment]
Hmmm, that's odd...for me in Chrome & Firefox it looks like this:
Screen Shot 2019-06-06 at 5 18 06 PM

**Comment by natalieworth** [comment]
Yeah, mine looks the same as @youriwims

File: templates/socialaccount/snippets/social-buttons.html [hunk 1]
```diff
@@ -23,10 +23,10 @@
   {% endif %}
   {% is_next_url_admin as show_superuser_login %}
   {% if show_superuser_login and auth_process == 'login' %}
-  <a href="{% url 'admin:login' %}">
-    <button class="btn social-admin">
+  <a href="{% url 'admin:login' %}" class="d-none">
```
Comment (gideonthomas) [inline templates/socialaccount/snippets/social-buttons.html]: @youriwims why is this set to `display: none`?

Comment (youriwims) [reply templates/socialaccount/snippets/social-buttons.html]: It was primarily for design review. I can revert it now.

Comment (gideonthomas) [reply templates/socialaccount/snippets/social-buttons.html]: yeah revert it. the only way to login in to review apps is using super users which means that this button needs to be there for testing.

For future reference as well: For design review, if they want to see the pages without the django admin button, you just need to change the url in the `next` query string. For example: https://network-pulse-api-stagi-pr-505.herokuapp.com/accounts/login/?next=/api/pulse/v2/entries/

**Comment by natalieworth** [comment]
Hey @youriwims , sorry, I was using the old link colours on this page. Could we change the links at the bottom to the dark blue?

**Comment by youriwims** [comment]
Yep, just did!

**Comment by gideonthomas** [comment]
@natalieworth let me know once design review is complete. I think the pages needed to be tested are:
Login: network-pulse-api-stagi-pr-505.herokuapp.com/accounts/login/?next=/api/pulse/v2/entries
Email Exists: network-pulse-api-stagi-pr-505.herokuapp.com/accounts/login/?next=%2Faccounts%2Fsocial%2Fconnections%2F%3Fnext%3D%252Fadmin%252F&promptconnection=True&provider=GitHub

**Comment by youriwims** [comment]
Sorry, @natalieworth, it looks like there's another screen. You can view it here: https://network-pulse-api-stagi-pr-505.herokuapp.com/accounts/github/login/callback/. 😬 🙃 
cc @gideonthomas
Screen Shot 2019-06-11 at 2 38 01 PM

**Comment by gideonthomas** [comment]
@youriwims, @natalieworth don't worry about that screen, it should really never show up to a user, it shows up for you because we didn't setup social auth completely for your review app.

File: templates/socialaccount/snippets/social-buttons.html [hunk 2]
```diff
@@ -24,9 +24,9 @@
   {% is_next_url_admin as show_superuser_login %}
   {% if show_superuser_login and auth_process == 'login' %}
   <a href="{% url 'admin:login' %}">
-    <button class="btn social-admin">
+    <button class="btn social-admin pr-3">
       <img class="social-icon" src="{% static 'account/img/admin-icon.svg' %}">
-      {% trans "Sign in with Django Admin" %}
+      {% trans "Sign in with Django <br> Admin" %}
```
[the remark this answers is not in the document]
Comment (youriwims) [reply templates/socialaccount/snippets/social-buttons.html]: This was just decision based on aesthetic, I can remove it.

File: staticfiles/account/css/login.css [hunk 3]
```diff
@@ -2,29 +2,24 @@
...
 }
 
 .social-buttons .btn {
-  font-family: "Open Sans", sans-serif;
-  font-weight: 600;
+  font-weight: 700;
   font-size: 1rem;
+  line-height: 1.25;
   padding: 0;
-  width: 100%;
```
[the remark this answers is not in the document]
Comment (youriwims) [reply staticfiles/account/css/login.css]: added a max-width

File: staticfiles/account/css/login.css [hunk 4]
```diff
@@ -2,29 +2,24 @@
...
-}
-
 .social-buttons a {
   text-decoration: none;
 }
 
 .social-buttons .btn {
-  font-family: "Open Sans", sans-serif;
-  font-weight: 600;
+  font-weight: 700;
```
[the remark this answers is not in the document]
Comment (youriwims) [reply staticfiles/account/css/login.css]: that was an accident—thanks for catching it.

File: templates/base.html [hunk 5]
```diff
@@ -19,9 +21,9 @@
           <p class="main-title">{% block main_title %}{% trans "Welcome to the Mozilla Pulse API Service" %}{% endblock %}</p>
           {% block review_app_warning %}
           {% is_review_app as review_app %}
-          {% if review_app %}
+          <!-- {% if review_app %}
```
Comment (gideonthomas) [inline templates/base.html]: revert

File: templates/socialaccount/snippets/social-buttons.html [hunk 6]
```diff
@@ -3,18 +3,18 @@
 {% is_review_app as review_app %}
 <div class="social-buttons">
   {% with auth_process=process|default:"login" excluded_provider=exclude|lower %}
-  {% if not review_app %}
+  {% if review_app %}
```
Comment (gideonthomas) [inline templates/socialaccount/snippets/social-buttons.html]: revert

**Pull request merged**